code/index.html

```diff
@@ -5,33 +5,82 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="style.css" />
```

Comment:
i just learned that we should add './' before the 'style.css' just to make it consistent while adding link. Good habit to start with. :-)